app/src/components/FileList.svelte

```diff
@@ -0,0 +1,172 @@
+<script>
+  import ApolloClient from "apollo-boost";
+  import { gql } from "apollo-boost";
+  import { getClient, query } from "svelte-apollo";
+
+  import { format } from "timeago.js";
+
+  import { getContext } from "svelte";
+  import { revision, objectPath } from "../stores.js";
+  import * as path from "../path.js";
+
+  import { Caption, Icon, Text } from "../DesignSystem";
+  import CommitTeaser from "../components/CommitTeaser.svelte";
+  import { link } from "svelte-spa-router";
+
+  const TREE = gql`
```

Comment:
These start to pop up more and more and live inline, would it make sense to centralise our queries somewhere? And as a follow-up start validating.

Reply:
As discussed IRL: we'll leave them where they are for now, because they're very specific to the views.